src/popup.tsx

```
@@ -763,6 +765,15 @@ const PopupDefaultExample = ({
           {/* Accounts tab */}
           {settingsTab === 'accounts' && (
           <div style={{ padding: '4px 0' }}>
+            <div style={{ padding: '2px 16px 6px', fontSize: '11px', color: THEME.text.secondary }}>
```

ESLint

[error] Replace `<div·style={{·padding:·'2px·16px·6px',·fontSize:·'11px',·color:·THEME.text.secondary·}}` with `··<div⏎················style={{⏎··················padding:·'2px·16px·6px',⏎··················fontSize:·'11px',⏎··················color:·THEME.text.secondary,⏎················}}⏎··············`

(prettier/prettier)